docs/wiki/NuGet-packages.md

```diff
@@ -13,6 +13,40 @@ This package is primarily intended as a method for rapidly shipping hotfixes to
 Note: this package is intended as a replacement for Microsoft.Net.Compilers (which is a Windows-only package) and Microsoft.NETCore.Compilers. Those packages are now deprecated and will be deleted in the future.
 
 ## Versioning
+Starting on Roslyn versions `5.0` and above, Roslyn NuGet packages are shipped with the .NET SDK instead of VS.  New Roslyn packages will be published on every SDK preview and GA release.
```

Comment:
The title of the PR says "_move_ package publishing out of Roslyn official builds" but as far as I understand, we already publish Roslyn in the VMR today (resulting in `.1xx` versions). We also publish it from Roslyn official builds (`.x` versions). So the PR is more about just removing the latter, right?

Reply:
Generally yes - and adjusting the version slightly to ensure VMR can produce non-conflicting stable packages.